src/app/components/Typography/Typography.module.css

```diff
@@ -0,0 +1,55 @@
+.xl {
+  font-family: var(--font-family-primary);
+  font-weight: var(--font-weight-regular);
```

Comment:
Any style properties that don't change, you might as well put in one class and chain them together with the changing class in the JSX